### PR TITLE
update the translation of `Object.fromEntries`

### PR DESCRIPTION
**Pull request by robert-jx**
### Description
update the translation of `Object.fromEntries`

### Motivation
update the translation of `Object.fromEntries`

### Additional details
none

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/object/fromentries/index.md [hunk 1]
```diff
@@ -5,63 +5,79 @@ slug: Web/JavaScript/Reference/Global_Objects/Object/fromEntries
 
 {{JSRef}}
 
-**`Object.fromEntries()`** 方法把键值对列表转换为一个对象。
+**`Object.fromEntries()`** 静态方法将键值对列表转换成对象。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/object/fromentries/index.md:8]:
```suggestion
**`Object.fromEntries()`** 静态方法将键值对列表转换为一个对象。
```

File: files/zh-cn/web/javascript/reference/global_objects/object/fromentries/index.md [hunk 2]
```diff
@@ -5,63 +5,79 @@ slug: Web/JavaScript/Reference/Global_Objects/Object/fromEntries
...
+```js-nolint
+Object.fromEntries(iterable)
 ```
 
 ### 参数
 
 - `iterable`
-  - : 类似 {{jsxref("Array")}} 、 {{jsxref("Map")}} 或者其他实现了[可迭代协议](/zh-CN/docs/Web/JavaScript/Reference/Iteration_protocols#The_iterable_protocol)的可迭代对象。
+
+  - : 一个[可迭代的](/zh-CN/docs/Web/JavaScript/Reference/Iteration_protocols#可迭代协议)对象，例如 {{jsxref("Array")}} 或者 {{jsxref("Map")}}，包含对象列表。每个对象都要有两个属性：
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/object/fromentries/index.md:22]:
```suggestion
  - : 一个[可迭代](/zh-CN/docs/Web/JavaScript/Reference/Iteration_protocols#可迭代协议)对象，例如 {{jsxref("Array")}} 或者 {{jsxref("Map")}}，包含了对象列表。每个对象都要有两个属性：
```

File: files/zh-cn/web/javascript/reference/global_objects/object/fromentries/index.md [hunk 3]
```diff
@@ -5,63 +5,79 @@ slug: Web/JavaScript/Reference/Global_Objects/Object/fromEntries
...
 
 ### 返回值
 
-一个由该迭代对象条目提供对应属性的新对象。
+一个新对象，其属性由可迭代对象的条目给定。
 
 ## 描述
 
-`Object.fromEntries()` 方法接收一个键值对的列表参数，并返回一个带有这些键值对的新对象。这个迭代参数应该是一个能够实现`@@iterator`方法的的对象，返回一个迭代器对象。它生成一个具有两个元素的类数组的对象，第一个元素是将用作属性键的值，第二个元素是与该属性键关联的值。
+`Object.fromEntries()` 方法获取一个键值对列表，并返回一个新对象，该对象的属性由这些条目给定。`iterable` 参数应是实现 `@@iterator` 方法的对象。该方法返回一个可迭代对象，该对象生成类似于两个元素数组的对象。第一个元素是将用作属性键的值，第二个元素是要与该属性键关联的值。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/object/fromentries/index.md:37]:
```suggestion
`Object.fromEntries()` 方法接收一个键值对列表，并返回一个新对象，该对象的属性由这些条目给定。`iterable` 参数应该是实现了 `@@iterator` 方法的可迭代对象。该方法返回一个可迭代对象，产生包含两个元素的类数组对象。第一个元素是将用作属性键的值，第二个元素是要与该属性键关联的值。
```

File: files/zh-cn/web/javascript/reference/global_objects/object/fromentries/index.md [hunk 4]
```diff
@@ -5,63 +5,79 @@ slug: Web/JavaScript/Reference/Global_Objects/Object/fromEntries
...
+const map = new Map([
+  ["foo", "bar"],
+  ["baz", 42],
+]);
 const obj = Object.fromEntries(map);
 console.log(obj); // { foo: "bar", baz: 42 }
 ```
 
-### `Array` 转化为 `Object`
+### 数组转换成对象
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/object/fromentries/index.md:58]: 最好保持一致性吧
```suggestion
### Array 转换成对象
```

File: files/zh-cn/web/javascript/reference/global_objects/object/fromentries/index.md [hunk 5]
```diff
@@ -5,63 +5,79 @@ slug: Web/JavaScript/Reference/Global_Objects/Object/fromEntries
...
+  ["2", "c"],
+];
 const obj = Object.fromEntries(arr);
 console.log(obj); // { 0: "a", 1: "b", 2: "c" }
 ```
 
 ### 对象转换
 
-`Object.fromEntries` 是与 {{jsxref("Object.entries()")}} 相反的方法，用 [数组处理函数](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Array#Methods_2) 可以像下面这样转换对象：
+通过 `Object.fromEntries`, 其反向方法 {{jsxref("Object.entries()")}}，和[数组操作方法](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Array#实例方法)，你可以像这样转换对象：
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/object/fromentries/index.md:74]:
```suggestion
你可以通过 `Object.fromEntries`、其逆操作 {{jsxref("Object.entries()")}} 和[数组操作方法](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Array#实例方法)，像这样转换对象：
```

**Review by JasonLamv-t — COMMENTED** [review]
Some suggestions.

File: files/zh-cn/web/javascript/reference/global_objects/object/fromentries/index.md [hunk 6]
```diff
@@ -5,63 +5,79 @@ slug: Web/JavaScript/Reference/Global_Objects/Object/fromEntries
...
+```js-nolint
+Object.fromEntries(iterable)
 ```
 
 ### 参数
 
 - `iterable`
-  - : 类似 {{jsxref("Array")}} 、 {{jsxref("Map")}} 或者其他实现了[可迭代协议](/zh-CN/docs/Web/JavaScript/Reference/Iteration_protocols#The_iterable_protocol)的可迭代对象。
+
+  - : 一个[可迭代](/zh-CN/docs/Web/JavaScript/Reference/Iteration_protocols#可迭代协议)对象，例如 {{jsxref("Array")}} 或者 {{jsxref("Map")}}，包含了对象列表。每个对象都要有两个属性：
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/object/fromentries/index.md:22]:
```suggestion
  - : 一个包含对象列表的[可迭代](/zh-CN/docs/Web/JavaScript/Reference/Iteration_protocols#可迭代协议)对象，例如 {{jsxref("Array")}} 或者 {{jsxref("Map")}}。每个对象都要有两个属性：
```

File: files/zh-cn/web/javascript/reference/global_objects/object/fromentries/index.md [hunk 7]
```diff
@@ -5,63 +5,79 @@ slug: Web/JavaScript/Reference/Global_Objects/Object/fromEntries
...
+
+    - `0`
+      - : 表示属性键的字符串或者 [symbol](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Symbol)。
+    - `1`
+      - : 属性值。
+
+    通常，该对象被实现为两个元素的数组，第一个元素是属性键，第二个元素是属性值。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/object/fromentries/index.md:29]: I prefer "二元数组", as "两个元素的数组" may be regarded as an array which has only two elements.

```suggestion
    通常，该对象被实现为二元数组，第一个元素是属性键，第二个元素是属性值。
```

image

File: files/zh-cn/web/javascript/reference/global_objects/object/fromentries/index.md [hunk 8]
```diff
@@ -5,63 +5,79 @@ slug: Web/JavaScript/Reference/Global_Objects/Object/fromEntries
...
 
-一个由该迭代对象条目提供对应属性的新对象。
+一个新对象，其属性由可迭代对象的条目给定。
 
 ## 描述
 
-`Object.fromEntries()` 方法接收一个键值对的列表参数，并返回一个带有这些键值对的新对象。这个迭代参数应该是一个能够实现`@@iterator`方法的的对象，返回一个迭代器对象。它生成一个具有两个元素的类数组的对象，第一个元素是将用作属性键的值，第二个元素是与该属性键关联的值。
+`Object.fromEntries()` 方法接收一个键值对列表，并返回一个新对象，该对象的属性由这些条目给定。`iterable` 参数应该是实现了 `@@iterator` 方法的可迭代对象。该方法返回一个可迭代对象，产生包含两个元素的类数组对象。第一个元素是将用作属性键的值，第二个元素是要与该属性键关联的值。
+
+`Object.fromEntries()` 执行 {{jsxref("Object.entries()")}} 的相反操作，只是 `Object.entries()` 只返回字符串键属性，而 `Object.fromEntries()` 还可以创建符号键属性。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/object/fromentries/index.md:39]: https://baike.baidu.com/item/%E9%80%86%E6%93%8D%E4%BD%9C/56176855

```suggestion
`Object.fromEntries()` 执行 {{jsxref("Object.entries()")}} 的逆操作，只是 `Object.entries()` 只返回字符串键属性，而 `Object.fromEntries()` 还可以创建符号键属性。
```

**Review by yin1999 — COMMENTED** [review]
Some suggestions. Sorry for the delay

**Review by yin1999 — COMMENTED** [review]
Thanks @robert-jx 💯

**Pull request merged**